crates/turborepo-lib/src/cli.rs

```diff
@@ -96,6 +93,34 @@ pub struct Args {
     pub command: Option<Command>,
 }
 
+#[derive(Debug, Parser, Serialize, Clone, Copy, PartialEq, Eq, Default)]
+#[serde(into = "u8")]
+pub struct Verbosity {
+    #[clap(
+        long = "verbosity",
+        global = true,
+        conflicts_with = "v",
+        value_name = "COUNT"
+    )]
+    /// Verbosity level
+    pub verbosity: Option<u8>,
+    #[clap(
+        short = 'v',
+        action = clap::ArgAction::Count,
+        global = true,
+        hide = true,
+        conflicts_with = "verbosity"
+    )]
+    pub v: u8,
+}
+
+impl Into<u8> for Verbosity {
```

Comment:
nit: clippy wants a `From` impl instead of an `Into`:

> warning: an implementation of `From` is preferred since it gives you `Into<_>` for free where the reverse isn't true
   --> crates/turborepo-lib/src/cli.rs:117:1
    |
117 | impl Into<u8> for Verbosity {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = help: `impl From<Local> for Foreign` is allowed by the orphan rules, for more information see
            https://doc.rust-lang.org/reference/items/implementations.html#trait-implementation-coherence
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#from_over_into
    = note: `#[warn(clippy::from_over_into)]` on by default
help: replace the `Into` implentation with `From<cli::Verbosity>`
    |
117 ~ impl From<Verbosity> for u8 {
118 ~     fn from(val: Verbosity) -> Self {
119 ~         let Verbosity { verbosity, v } = val;

Reply:
Agh, I thought I had clippy setup.